writing-visualising/dashboards_rshiny.qmd

```diff
@@ -7,36 +7,148 @@ title: "Creating R Shiny dashboards"
 
 ---
 
-# Learning R Shiny
+## Learning R Shiny
 
 ---
 
-Shiny is an R package that makes it easy to build interactive web apps straight from R. You can host standalone apps on a webpage, embed them in [R Markdown documents](http://rmarkdown.rstudio.com/) or build [dashboards](http://rstudio.github.io/shinydashboard/). You can also extend your Shiny apps with [CSS themes](http://rstudio.github.io/shinythemes/), [htmlwidgets](http://www.htmlwidgets.org/), and [JavaScript actions](https://github.com/daattali/shinyjs/blob/master/README.md).
+Shiny is an R package that makes it easy to build interactive web apps straight from R. You can host standalone apps on a webpage, embed them in [R Markdown documents](http://rmarkdown.rstudio.com/) or build dashboards. You can also extend your Shiny apps with [CSS themes](http://rstudio.github.io/shinythemes/), [htmlwidgets](http://www.htmlwidgets.org/), and [JavaScript actions](https://github.com/daattali/shinyjs/blob/master/README.md).
 
 The guidance on this page assumes you have prior knowledge of R, if you are new to R, then take a look at our [R resources page](../learning-development/r.html) to get started.
 
+For some initial inspiration, check out the [Shiny gallery](https://shiny.posit.co/r/gallery/).
+
+### Getting started with Shiny
+
 There are a lot of resources already available to support you when working with R Shiny, if you're new, then this [blog post](https://www.r-bloggers.com/2021/03/a-gentle-introduction-to-r-shiny/) provides a gentle introduction.
 
+If you like learning by reading/doing then the [Posit Shiny-specific site](https://shiny.posit.co/r/getstarted/shiny-basics/lesson1/index.html) has lots of resources that you can  work through. Likewise [Introduction section of Mastering Shiny](https://mastering-shiny.org/basic-intro.html) is also a helpful resources.
+
+If you like learning by watching then the [Posit Youtube channel](https://www.youtube.com/@PositPBC/videos) is also full of good resources. 
+
+### Creating a user interface
+
 Our [template shiny app repository](https://github.com/dfe-analytical-services/shiny-template) is a useful starting point for all public facing dashboards in the department. Please see the [dashboard template section](#dfe-dashboard-template) for guidance on what the dashboard template includes and how to use it.
```

Comment:
Chris didn't do this but the #dfe-dashboard-template link isn't working (think it needs changing to #dfe-styling-and-the-dfe-r-shiny-template ???)